swarm_copy/app/main.py

```diff
@@ -111,26 +129,56 @@ async def run_simple_agent(
     agent_routine: Annotated[AgentsRoutine, Depends(get_agents_routine)],
     agent: Annotated[Agent, Depends(get_starting_agent)],
     context_variables: Annotated[dict[str, Any], Depends(get_context_variables)],
-) -> list[Any]:
+    session: Annotated[AsyncSession, Depends(get_session)],
+    thread_id: Annotated[str, Depends(get_thread_id)],
+) -> AgentResponse:
```

Comment:
Not really the purpose of this MR, but now that the main endpoints are back do you think we could reintroduce the routers (i.e. not have these endpoints in main) ? If not feel free to create a ticket for it, or we will address in the CRUD ticket.